### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by warpech**
This is a first issue template in Juicy, so I'd like you @tomalec to review it. 

Then we can add the same one to other repos.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Steps to reproduce
+
+### Expected result
```
Comment (tomalec) [inline .github/ISSUE_TEMPLATE.md:7]: I would add 

### Live demo

with a link to jsbin/jsfiddle, we should prepare an example that uses polygit to load juicy components to be used for forking, as in https://github.com/Polymer/polymer/blob/master/.github/ISSUE_TEMPLATE.md#live-demo

**Comment by tomalec** [comment]
Here is the jsbin sample https://jsbin.com/ligimo/3/edit?html,output

**Comment by tomalec** [comment]
Here is my proposition, @warpech Please say WDYT

**Comment by warpech** [comment]
Perfect, thanks! Merging

**Pull request merged**